src/runtime/node/crypto/$cloudflare.ts

```diff
@@ -128,14 +128,14 @@ export default {
    */
   Certificate,
   Cipher,
-  // @ts-expect-error undocumented public API
```

Comment:
I don't know why this was needed before but now it's not needed. I suspect that this some kind of typescript bug. 🤷🏻